l10n_it_edi_related_document_type/models/account_move_line.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class AccountInvoiceLine(models.Model):
+    # _position = ['2.2.1']
+    _inherit = "account.move.line"
+
+    related_documents = fields.One2many(
```

Comment:
```suggestion
    related_document_ids = fields.One2many(
```
vedi https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#fields:
> One2Many and Many2Many fields should always have _ids as suffix (example: sale_order_line_ids)

Reply:
@matteonext non c'è più `account_move_line.py`? Come mai?

Reply:
Ah per questo https://github.com/OCA/l10n-italy/pull/4749#discussion_r2084850605

Reply:
Questo codice non esiste più quindi :+1: